service/src/main/kotlin/fi/espoo/evaka/messaging/bulletin/BulletinNotificationEmailService.kt

```diff
@@ -0,0 +1,169 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.messaging.bulletin
+
+import fi.espoo.evaka.daycare.domain.Language
+import fi.espoo.evaka.emailclient.IEmailClient
+import fi.espoo.evaka.shared.async.AsyncJobRunner
+import fi.espoo.evaka.shared.async.SendUnreadBulletinNotificationEmail
+import fi.espoo.evaka.shared.db.Database
+import mu.KotlinLogging
+import org.jdbi.v3.core.kotlin.mapTo
+import org.springframework.core.env.Environment
+import org.springframework.stereotype.Service
+import java.time.Instant
+import java.util.UUID
+
+private val logger = KotlinLogging.logger { }
+
+@Service
+class BulletinNotificationEmailService(
+    private val asyncJobRunner: AsyncJobRunner,
+    private val emailClient: IEmailClient,
+    env: Environment
+) {
+    init {
+        asyncJobRunner.sendBulletinNotificationEmail = ::sendBulletinNotification
+    }
+
+    val fromAddress = env.getRequiredProperty("mail_reply_to_address")
+
+    data class BulletinNotification(
+        val bulletinId: UUID,
+        val receiverId: UUID,
+        val receiverEmail: String,
+        val language: String?
```

Comment:
Why not map directly to `Language`? IMO it's better to map earlier rather than later to avoid mistakes with forgotten mappings.